src/parser/jobs/ast/modules/ArcanaSuggestions.js

```diff
@@ -56,7 +48,7 @@ export default class ArcanaSuggestions extends Module {
 
 		this.cardLogs = this.arcanaTracking.getCardLogs.map(artifact => {
 
-			const isArcana = artifact.lastEvent && [...DRAWN_ARCANA_USE, ...HELD_ARCANA_USE].includes(artifact.lastEvent.ability.guid)
+			const isArcana = artifact.lastEvent && [...PLAY].includes(artifact.lastEvent.ability.guid)
```

Comment:
```suggestion
			const isArcana = artifact.lastEvent && PLAY.includes(artifact.lastEvent.ability.guid)
```

`.includes` doesn't modify the array, so don't need to copy it